### ISSUE TITLE
Hello, I have some questions regarding the Occ3D-nuScenes dataset.

### ISSUE THREAD

**Issue by yueyangwen** [lzzzzzm/STCOcc#15]
On the Occ3D-nuScenes website（https://github.com/Tsinghua-MARS-Lab/Occ3D?tab=readme-ov-file#occ3d-nuscenes）, I noticed that the Occ3D-nuScenes dataset has ​600 scenes​ for training, ​150 for validation, and ​200 for testing​ (totaling 950).

​First question:​​
Your gts folder only contains ​850 scenes, not 950. Why is this the case?

​Second question:​​
The results folder in your test code only has ​150 scenes. Does this mean you tested on the ​validation set​ instead of the ​test set​?

Could you please clarify these two points? Thank you very much!
